typings/index.test-d.ts

```diff
@@ -1062,3 +1065,71 @@ collector.on('end', (collection, reason) => {
 });
 
 expectType<Promise<number | null>>(shard.eval(c => c.readyTimestamp));
+
+// Test audit logs
+declare const guild: Guild;
+expectType<Promise<GuildAuditLogs<'MEMBER_KICK'>>>(guild.fetchAuditLogs({ type: 'MEMBER_KICK' }));
+expectType<Promise<GuildAuditLogs<20>>>(guild.fetchAuditLogs({ type: GuildAuditLogs.Actions.MEMBER_KICK }));
```

Comment:
Are we not able to use the enums from discord-api-types? At least to avoid using magic numbers.

Reply:
Fixed in a250ec7 !

Reply:
Uh-oh, forgot to run the tests, but they don't pass anymore...

```
Parameter type Promise<GuildAuditLogs<AuditLogEvent.MemberKick>> is not identical to argument type Promise<GuildAuditLogs<20>>
```
But `AuditLogEvent.MemberKick` *is* 20